app/src/main/java/com/example/dynamicdiet/MainActivity.kt

```diff
@@ -95,6 +95,21 @@ fun SimpleFilledTextFieldSample() {
     }
 }
 
+@Preview(showBackground = true)
+@Composable
+fun DefaultPreview(){
+    DynamicDietTheme{
+        Surface(
+            color = MaterialTheme.colors.background,
+            modifier = Modifier.fillMaxWidth()
+        ) {
+            Greeting("Justin")
+            SimpleFilledTextFieldSample()
```

Comment:
I have a hunch these are practice/dummy functions at this point.  Refactoring doesn't add much value.  Consider if these functions are needed in the production application.  If not, remove/replace them.